features/git-town-new-pull-request/self_hosted.feature

```diff
@@ -0,0 +1,45 @@
+Feature: git-town new-pull-request: when origin is a self hosted servie
+
+  As a developer trying to create a pull request when using a self hosted service
+  I want to be able to configure git town with the code hosting driver it should use
+  So that new-pull-request works with my self hosted service
+
+
+  Background:
+    Given I have "open" installed
+
+
+  Scenario: self hosted bitbucket
+    Given I have a feature branch named "feature"
+    And my remote origin is "git@self-hosted-bitbucket:Originate/git-town.git"
```

Comment:
Let's use a URL that does not give away the repo type here, e.g. `git@repo:Originate/git-town.git`